src/sensai/featuregen.py

```diff
@@ -269,6 +278,17 @@ def _generate(self, df: pd.DataFrame, ctx=None) -> pd.DataFrame:
         return resultDf
 
 
+def getFlattenedFeatureGenerator(fgen: FeatureGenerator, ruleTemplate: data_transformation.DFTNormalisation.RuleTemplate):
+    """
+    Return a feature generator that will generate flattened output of the provided fgen
+
+    :param fgen: feature generator to be flattened
+    :param ruleTemplate: ruleTemplate to apply to all output columns
```

Comment:
The parameter should always be called "normalisationRuleTemplate" to make its purpose clear.

Reply:
Done